src/services/Storage.ts

```diff
@@ -1,26 +1,34 @@
-import { noteToActivityPub } from "../utilities/activityPub";
+import {
+  noteToActivityPub,
+  postReplyToActivityPub,
+} from "../utilities/activityPub";
 import { FeedItem, HexString } from "../utilities/types";
-import { ActivityPub } from "../utilities/activityPubTypes";
 import {
   Content,
   WriteStreamCallback,
   WriteStream,
   StoreInterface,
 } from "@dsnp/sdk/core/store";
 import { isFunction, isUint8Array } from "./utilities";
+import { NoteActivityPub } from "../utilities/activityPubTypes";
 
 export const createNote = async (
   actor: HexString,
   note: string,
   uriList: string[]
-): Promise<FeedItem> => {
+): Promise<FeedItem<NoteActivityPub>> => {
   // send content to api
-  const activityPubNote: ActivityPub = noteToActivityPub(actor, note, uriList);
-  const newPostFeedItem: FeedItem = {
+  const activityPubNote: NoteActivityPub = noteToActivityPub(
+    actor,
+    note,
+    uriList
+  );
+  const newPostFeedItem: FeedItem<NoteActivityPub> = {
     fromAddress: actor,
     content: activityPubNote,
     blockNumber: 0x123,
     timestamp: Math.floor(Math.random() * 999999),
+    hash: "0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7",
```

Comment:
**going to move this into a new story because we are currently not fetching the data from the api endpoint, we're just mocking it here.  I don't want to add any more work to this story for the sake of not making it bigger than it already is.